### PR TITLE
Descriptive 404

### PR DESCRIPTION
**Pull request by akshaykadambatt**

### CODE REVIEW

**Comment by MartijnCuppens** [comment]
I think we can do better than that. It would be nice if we can create something unique.

**Comment by akshayknz** [comment]
Like a new GUI page?

**Comment by MartijnCuppens** [comment]
Yeah, and maybe we could add the doc search to the page

**Comment by akshayknz** [comment]
Well, you might have noticed. I know nothing about git. From the second you said "doc search", I was searching the whole internet googling "how to add doc search in GitHub".

File: site/content/404.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ sitemap_exclude: true
 <div class="text-center py-5">
   <h1 class="display-1">404</h1>
   <h2>File not found</h2>
+  <p> The file/ path you are looking for is not found.</p>
```
Comment (XhmikosR) [inline site/content/404.md]: Extra space after `<p>` BTW

**Comment by XhmikosR** [comment]
I'm gonna close this. The 404 page needs a redesign.

**Pull request closed without merging**